src/main/kotlin/no/nav/syfo/handlestatus/HandleStatusInvalid.kt

```diff
@@ -126,6 +130,8 @@ fun handlePatientNotFoundInPDL(
     log.info("Apprec Receipt sent to {}, {}", env.apprecQueueName, fields(loggingMeta))
 
     INVALID_MESSAGE_NO_NOTICE.inc()
+    duplicationCheckService.persistDuplicationCheck(duplicationCheckModel)
```

Comment:
Hadde det gått an å gjøre sånn som i syfosmmottak i stedet, der vi oppdaterer til slutt i prosesseringen i blockingapplicationrunner? Da slipper vi å gjøre dette så mange steder :)

Reply:
Eg tror vi oppdaterer den mange plasser i syfosmmottak også, det er i dei tilfellene, vi har ein "hopper" ut av loppen ` continue@loop` , så tror eg vi må oppdatere inni disse metodene

Reply:
Det er forsåvidt riktig, men når vi hopper ut av loopen så har vi en egen metode for det med å sende apprec, og der gjør vi dette for alle: https://github.com/navikt/syfosmmottak/blob/master/src/main/kotlin/no/nav/syfo/handlestatus/HandleStatusInvalid.kt#L717

Kanskje vi kunne hatt en tilsvarende metode som gjør det som er felles for alle reglene som kan slå ut her også? :)

Reply:
Ja det hadde ikkje vært så dumt

Reply:
Da skal det ha blitt flyttet